gui/canvaspainter/v7/qt5/rootqt5.cpp

```diff
@@ -0,0 +1,275 @@
+/****************************************************************************
+**
+** Copyright (C) 2016 The Qt Company Ltd.
```

Comment:
In principle, qt requires to keep there license statement.
From other side, beside few standard construction I fully change code of original qt example.
Therefore I fully remove qt copyright here